client/additional-methods-setup/upe-preview-methods-selector/enable-upe-preview-task.js

```diff
@@ -32,10 +32,12 @@ const EnableUpePreviewTask = () => {
 		<WizardTaskItem
 			title={ interpolateComponents( {
 				mixedString: __(
-					'Enable the new WooCommerce Payments checkout experience {{earlyAccessWrapper}}Early access{{/earlyAccessWrapper}}',
+					'{{wrapper}}Enable the new WooCommerce Payments checkout experience{{/wrapper}} ' +
+						'{{earlyAccessWrapper}}Early access{{/earlyAccessWrapper}}',
 					'woocommerce-payments'
 				),
 				components: {
+					wrapper: <span />,
```

Comment:
Why do we need a wrapper on top?

Reply:
Nvm, mis-read. Thought this is a wrapper on top of the `<Pill>`

Reply:
Good question! I thought it would make things a bit better by helping identify which items need to be vertically aligned, when using the `display: flex; align-items: center;` container that I added around the title

![2021-07-23 13 43 12](https://user-images.githubusercontent.com/273592/126827295-bad149d6-cf72-4ad8-bc10-c0336d6d2d0f.gif)